test-app/tests/unit/dom/fill-in-test.js

```diff
@@ -16,16 +16,6 @@ if (isFirefox || isChrome) {
   clickSteps.push('selectionchange');
 }
 
-/**
- * Prior to Chrome 129 (canary),
- * Chrome 127.x emits an extra selectionchange event sometimes
- *
- * Delete this once we don't want to test against Chrome 127
- */
-if (isChrome) {
-  clickSteps.push('selectionchange');
-}
```

Comment:
This is unrelated to the context reuse issue but fixes test failures currently on master. Since tests are running on Chrome 130 now, I assume this is safe to remove.